docs/src/checkpointer.md

```diff
@@ -127,11 +127,38 @@ Types to watch for:
 - `TimeVaryingInputs` (because they contain `NCDatasets`, which contain pointers
   to files)
 
+!!! Adapt and references
```

Comment:
It looks like this doesn't render correctly

<img width="823" height="168" alt="Image" src="https://github.com/user-attachments/assets/f9fa57b3-b29c-43fd-8d95-604c0c66ad4c" />

we could do:

```suggestion
!!! warning "Adapt and references"
```